### DISCUSSION TITLE
Local echo during serial terminal connection

### DISCUSSION THREAD

**Discussion by fredhim** [wezterm/wezterm#1768] [Q&A]
Hi.
I'm very impressed by the many configuration options of wezterm.

Is there a simple way to open a serial terminal with local echo,
so that I can see what I typed and has been sent out?

At the moment I am using:
wezterm serial --baud 57600 COM3

Maybe there is even a command line option for local echo (e.g., by using --config)?

Thank you!
Fred

**Comment by wez** [comment]
There isn't an option for this; would you mind opening an enhancement issue so that it doesn't get forgotten?
